lessons/python/scikit-learn/lesson.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+title: Machine Learning with scikit-learn
+visible: true
+tags:
+	- python
```

Comment:
Jekyll does not understand the tabs here and doesn't render the lesson; please replace with spaces.